pipeline/backend/local/local.go

```diff
@@ -257,7 +257,9 @@ func (e *local) getState(taskUUID string) (*workflowState, error) {
 	if !ok {
 		return nil, ErrWorkflowStateNotFound
 	}
-	return state.(*workflowState), nil
+
+	workflowState, _ := state.(*workflowState)
```

Comment:
if we ignore the ok here we should add a comment that we always only expect this to work

Reply:
Just added error handling.